azure-pipelines.yml

```diff
@@ -10,8 +10,11 @@ pool:
   vmImage: 'ubuntu-latest'
 
 variables:
-  buildConfiguration: 'Release'
-  BROWSER: chrome
+  - name: buildConfiguration
+    value: 'Release'
+  - name: BROWSER
+    value: chrome
+  - group: dev-secrets
```

Comment:
so it looks like we're pulling the secrets from the dev-secrets thing, is this a pipeline thing or a kubernetes thing and how do i set the values when running locally? just override the appsettings.json?

Reply:
That's a 'variable group' and you can find it in AzDo under: Pipelines > Library.

When you're running locally, you can either override the appsettings.json (and make sure not to commit those changes) or you can set env vars that will override them. I think the readme has a section on how to do that. Just note that in order to override a value that's nested - the server url for example, you need to name the var like so: PARENTKEY__CHILDKEY - for more examples have a look at the 'runTests' task in azure-pipelines.yaml